nectar/src/command/trade.rs

```diff
@@ -137,7 +137,7 @@ async fn init_maker(
 
     let spread: Spread = settings.maker.spread;
 
-    let strategy = strategy::AllIn::new(btc_fee_reserve, btc_max_sell, dai_max_sell, spread);
+    let strategy = strategy::AllIn::new(btc_fee_reserve, max_sell, max_buy, spread);
```

Comment:
Please, I have dealt with enough merge conflicts can you hold on with settings changes?

Reply:
If this comment is just concerning this file, and this specific rename I think I can remove it and add a ToDo for later. 
I was hoping to change all the names consistently at once, but if it is causing too much friction I can remove that one. 
`dai_max_sell` is misleading, because the value is a BTC value now (that represents the maximum buy quantity).

Reply:
Added `no-mergify`, going in after your PR has merged.